libnavigation-ui/src/main/res/values/styles.xml

```diff
@@ -143,24 +143,24 @@
             @color/mapbox_navigation_view_color_progress_background_dark
         </item>
 
-        <item name="navigationViewRouteStyle">@style/NavigationMapRoute</item>
+        <item name="navigationViewRouteStyle">@style/MapboxStyleNavigationMapRoute</item>
 
-        <item name="navigationViewLocationLayerStyle">@style/NavigationLocationLayerStyle</item>
+        <item name="navigationViewLocationLayerStyle">@style/MapboxStyleNavigationLocationLayerStyle</item>
 
-        <item name="navigationViewDestinationMarker">@drawable/map_marker_dark</item>
+        <item name="navigationViewDestinationMarker">@drawable/mapbox_ic_map_marker_dark</item>
```

Comment:
Could we convert this into a Vector Drawable so we can remove the PNGs? cc @map-angela

Reply:
I am going to fix this after https://github.com/mapbox/mapbox-navigation-android/issues/3459 is been addressed.